---
title: "Analysis Report 2: Influences of Non-small Cell Lung Cancer on Lipid Metabolism"
author: "Allison Bogisich"
date: "November 15, 2017"
output: github_document
bibliography: references.bib
csl: bioinformatics.csl
---

*Overall, a single-spaced page is about 500 words. So if the guidelines say half of a page, think about writing around 250 words. You can use the wordcountaddin in RStudio to track your progress.*

# Introduction

Add about 1 page here. Must cite at least 5 peer reviewed articles.

One of the most longstanding symptoms of cancer has been increased rates of lipid synthesis, as
cancer cells are prone to exhibit specific alterations in their metabolic activities. Cancer is
predicted to kill approximately 595,690 Americans in 2016, equating to about 1,630 people per
day according to the American Cancer Society. Cancer-associated metabolic reprogramming
occurs in order to support the increased production of cells via necessary increased rates of
biosynthesis of proteins, nucleic acids, and aforementioned lipids [@baenke2013hooked]. However,
not until relatively recently has research delving into the metabolic processes involved in lipid
biosynthesis and their regulation- within the context of human disease- sustained much traction.
The unique connection between lipid metabolism and cancer in humans lies within its linkage to
epidermal growth factor (EGF) proteins and their receptors [@normanno2006epidermal]. Non-small
cell lung cancer for example, is EGF specific and characterized by dramatic changes specifically
in phospholipid profiles [@marien2015non].There are many human lipid metabolism disorders
associated with aberrant lipid homeostasis besides cancers, including but not limited to:
Gaucher’s disease, Faber’s disease, Wolman’s disease and many others [@saltiel2001insulin, @madhusudan2004multicenter,@chanarin1975neutral]. While these diseases are fairly rare, there are
few current options for treatment and the survival rates-with decent quality of life-are quite low.

SNORAs and SNORDs are small noncoding RNAs involved in RNA processing and nogalamycin biosynthesis. This biosynthesis of noglamycin is critical to antirumor and antibiotic functionality of cellular maintenance. Both types have been found to be dysregulated in various types of cancer and have been implicated in not only the development but also the progressions of human cancer malignancy. Thus, I hypothesize that the expression of SNOR genes in the patient tissues examined by Li and Seo et al. will show a markeded flux in their levels of expression. That in the patients will later stage cancers, there will be more expression relative to the earlier stage cancer patients. I also predict that the smoking patients will have a higher degree of abberant expression than that of the non-smokers.

# Methods

## Sample origin and sequencing

Add about half a page here. In this section instead of first person (I/we), use Li et al. [@li2015rna] and Seo et al. [@seo2012transcriptional], since you'll just be describing what they did, based on the methods in their paper(s). It's important to include this so the reader knows what the study design was for the data you will be presenting.

## Computational

These are the methods that were used to process the sequencing data. Should probably be at least a half of a page. At a very minimum should include citations for biomartr, trimmomatic, and sailfish. Note that these three methods references don't count towards the five references you need to cite in the introduction.

# Results

In addition to a minimum of 4-5 figures/tables (and associated captions), you should include sufficient text in this section to describe what your findings were. Remember that in the results section you just describe what you found, but you don't interpret it - that happens in the discussion.

```{r load-libraries, message = FALSE, echo = FALSE}
# Be sure to install these packages before running this script
# They can be installed either with the intall.packages() function
# or with the 'Packages' pane in RStudio

# load general-use packages
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")
library("magrittr")

# this package allows for the easy inclusion of literature citations in our Rmd
# more info here: https://github.com/crsh/citr
# and here:
# http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
library("citr")
```

```{r load-data, message = FALSE, echo = FALSE}
# load the dataset from a compressed binary file
# it gets loaded as an object called "final_table"
# this has 3.4 million rows...so you will need to be thoughtful about
# how you analyze the data so that you don't overwhelm your laptop
load("output/final_compiled_counts/joined_count_data.RData")

# changes all columns names to lower case so Travis doesn't complain
names(final_table) %<>% tolower

# test that it loaded correctly before proceeding
stopifnot(exists("final_table"))
```

```{r make-summary-table, echo = FALSE}
# There are many many genes in this dataset, so we can
# subset it down to just a few here to look for interesting patterns
# in the most highly expressed
top_15 <- final_table %>%
  group_by(gender, genename) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count)) %>%
  head(n = 15)

# then we can use the `kable()` function to make a nicely formatted
# markdown table
top_15 %>%
  kable()
```

**Table 1**: The most highly expressed genes in both genders included *SFTPB* and *EEF1A1*.

```{r make-barplot-of-highly-expressed-genes, echo = FALSE}
# this code uses the same data as above, but use it to make a
# barplot - remember geom_col() is just like
# when you use geom_bar(stat = "identity")
top_15 %>%
  ggplot(aes(x = genename,
             y = mean_count,
             fill = gender)) +
    geom_col(position = "dodge")
```

**Figure 1**: Here we show an example figure caption.

```{r make-boxplot-of-highly-expressed-genes, echo = FALSE}
# here we just want to pull out the unique gene names and turn
# them into a vector so we can use it below to make a boxplot
# we use the pull() funtion to get this as a vector, just like
# we did when making histograms several weeks ago
top_genes <- top_15 %>%
  ungroup() %>%
  select(genename) %>%
  unique() %>%
  pull()

# now we need to filter from the full data set again, because
# we don't just want summary data, we want all the data in
# order to make boxplots
final_table %>%
  filter(genename %in% top_genes) %>%
  ggplot(aes(x = genename,
             y = counts_lengthscaledtpm,
             fill = gender)) +
    geom_boxplot() +
    facet_wrap(~cancer_stage) +
    xlab("Gene Name") +
    ylab("Scaled read counts per gene") +
    ggtitle("Read counts per gene by gender and cancer stage") +
    theme_bw() + # simplifies theme
    theme(axis.text.x = # rotates x axis labels vertically
            element_text(angle = 90,
                         hjust = 1))
```

**Figure 2**: Here we show another example figure caption.

```{r make-summary-table-SNOR-top15-genes}
#here I want to pull out just the top 15 genes that either 
#have SNORA- or SNORD- in their base name 
#have to filter it out as a subset from final_table
snor_genes <- final_table %>%
  filter(grepl("SNOR", genename)) %>%
  group_by(genename) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count)) %>%
  head(15)

# then we can use the `kable()` function to make a nicely formatted
# markdown table
snor_genes %>%
  kable()
```



```{r make-subset-SNOR-top50-genes}
#here I wantto pull out just the top 15 genes that either have SNORA- or SNORD- in their base name 
#have to filter it out as a subset from final_table
snor_genes_50 <- final_table %>%
  filter(grepl("SNOR", genename)) %>%
  group_by(genename) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count)) %>%
  head(50)

# then we can use the `kable()` function to make a nicely formatted
# markdown table
snor_genes_50 %>%
  kable()
```
```{r make-subset-SNOR-genes-all}
#here I wantto pull out all genes that have SNORA- or SNORD- in their base name 
#have to filter it out as a subset from final_table
total_snor <- final_table %>%
  filter(grepl("SNOR", genename)) %>%
  group_by(genename) %>%
  
#then if I try to make a table of it using 'kable'...
total_snor %>%
  kable()
```
 


# Discussion

Add around 1-2 pages interpreting your results and considering future directions one might take in analyzing these data.

# Sources Cited
