---
title: "Analysis Report 2: Influences of Non-small Cell Lung Cancer on Lipid Metabolism"
author: "Allison Bogisich"
date: "November 15, 2017"
output: github_document
bibliography: references.bib
csl: bioinformatics.csl
---

*Overall, a single-spaced page is about 500 words. So if the guidelines say half of a page, think about writing around 250 words. You can use the wordcountaddin in RStudio to track your progress.*

# Introduction

One of the most longstanding symptoms of cancer has been increased rates of cellular structure synthesis, as cancer cells are prone to exhibit specific alterations in their metabolic activities. Cancer is predicted to kill approximately 595,690 Americans in 2016, equating to about 1,630 people per
day according to the American Cancer Society. Cancer-associated metabolic reprogramming
occurs in order to support the increased production of cells via necessary increased rates of
biosynthesis of proteins, nucleic acids, and aforementioned lipids [@baenke2013hooked]. However,
not until relatively recently has research delving into the metabolic processes involved in lipid
biosynthesis and their regulation- within the context of human disease- sustained much traction.
The unique connection between lipid metabolism and cancer in humans lies within its linkage to
epidermal growth factor (EGF) proteins and their receptors [@normanno2006epidermal]. Non-small
cell lung cancer for example, is EGF specific and characterized by dramatic changes specifically
in phospholipid profiles [@marien2015non]. In 2017, non-small cell lung cancer is the most prevalent type of lung cancer, accounting for about 85 % of all lung cancer cases in the USA according to the American Cancer Society.There are many human lipid metabolism disorders associated with aberrant lipid homeostasis besides cancers, including but not limited to:Gaucher’s disease, Faber’s disease, Wolman’s disease and many others [@saltiel2001insulin, @madhusudan2004multicenter,@chanarin1975neutral]. While these diseases are fairly rare, there are few current options for treatment and the survival rates-with decent quality of life-are quite low.


SNORAs and SNORDs are small noncoding RNAs involved in RNA processing and nogalamycin biosynthesis. This biosynthesis of noglamycin is critical to antitumor and antibiotic functionality of cellular maintenance. Both types have been found to be dysregulated in various types of cancer and have been implicated in not only the development but also the progressions of human cancer malignancy. This additional analysis will be based on data derivied from the previous work of Li and Seo et al. in non-small cell cancer patients. I hypothesize that the expression of SNOR genes in the patient tissues examined by Li and Seo et al. will show a markeded flux in their levels of expression. That in the patients will later stage cancers, there will be more expression relative to the earlier stage cancer patients. I also predict that the smoking patients will have a higher degree of abberant expression than that of the non-smokers, due to the additional carcinogens found in tobacco products [@freour2008active].

# Methods

## Sample origin and sequencing

The work of Li and her team [@li2015rna] included data from the first large scale RNA sequencing study of lung adenocarcinoma by Seo et al. [@seo2012transcriptional]. Paired surgical samples of both cancerous lung tissue and non-cancerous tissue were taken from 200 Korean patients. Only eighty-seven samples were then sequenced using RNA-seq and the Illumina TruSeq SBS Kit v3 on a HiSeq 2000 sequencer (Illumina Inc.) to obtain 100-bp paired-end reads. Of the 87 individuals whose cancer specimens were screened and sequenced, smoking history prior to diagnosis of lung cancer was provided by only 83 (47 smokers, 36 never-smokers, and four of unknown history).They were then aligned and read based on the NCBI human reference genome assembly build (build 37.1). Li et al. then downloaded Seo et al.'s RNA sequencing and phenotypic metadata for 68 patients(with 34 validated smoking and 34 non-smoking histories) from Gene Expression Omnibus (GEO) with accession number GSE40419. Comparisons were conducted based on phenotypes, including age at diagnosis,gender, and smoking history which are available from the public website (http://genome.cshlp.org/content/22/11/2109/suppl/DC1). Additionally, in order to validate genes identified in the nonsmoker group, independent RNA-seq data from six non-smoker patients were also downloaded from another GEO with accessionnumber GSE37765 [@kim2013high]. Differential gene expression analysis was then conducted between paired normal and tumor tissue using R Bioconductor and edgeR. Genotype calling was done for single-nucleotide variants, from RNA-seq bam files, with pair-end RNA-seq reads aligned to humangenome assembly Ensembl GRCh37 by Tophat.

## Computational

In order to analyze the data from Li et al., a series of both R and bash scripts were run. In the first script, the NCBI data set downloaded (using 'aspera' and 'sra-toolkit' for added effiency) can be found here:http://ftp-trace.ncbi.nlm.nih.gov/sra/sdk/current/sratoolkit.current-ubuntu64.tar.gz. The second script facilitated the downloading of the *Homo sapien* genome via the 'Biomartr' pacakge (Drost and Paszkowski,2017). Output was then piped into 'Ref-seq'so that we could quickly aquire all the reference genome (RNA and annotations). Subsequently, the third script contains the export path and simple for-loop that converted .sra to .fastq format and put it into our output directory. Next in the pipe was converting sequences to k-mers and building an index to map against the original reference DNA for use with 'Sailfish'[@patro2014sailfish]. Then a large amount of time was alloted for running fastqc on all fastq files and saving output to output dirrectory, using 86 threads. Once complete, the large data set was trimmed to one of a more manageable size using 'trimmomatic' on paired-end DNA sequences in order to get rid of Illumina tags, gummy ends, suspicious quality sequences [@bolger2014trimmomatic]. Code checks were run to ensure completion of trimming step prior to running of 'Sailfish', which trimmed out base names while reading and running prior to output. All aforementioned steps were completed in TMUX, and all subsequently described steps were done in R 3.4.2 and R stuio using version 1.1.383. The final script consolidated counts by matching transcript identifiers to the gene name and pooling them all together at the gene level prior to counting the number of reads that have mapped to the gene. The output table was then metled with the metadata table to create our final_table for use in exploratory data analysis using 'tidyr', in order to create more data parsing friendly long-form data . The 'ggplot2' package could then be used for visualization of data into graphic results.


# Results

In addition to a minimum of 4-5 figures/tables (and associated captions), you should include sufficient text in this section to describe what your findings were. Remember that in the results section you just describe what you found, but you don't interpret it - that happens in the discussion.

```{r load-libraries, message = FALSE, echo = FALSE}
# Be sure to install these packages before running this script
# They can be installed either with the intall.packages() function
# or with the 'Packages' pane in RStudio

# load general-use packages
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")
library("magrittr")

# this package allows for the easy inclusion of literature citations in our Rmd
# more info here: https://github.com/crsh/citr
# and here:
# http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
library("citr")
```

```{r load-data, message = FALSE, echo = FALSE}
# load the dataset from a compressed binary file
# it gets loaded as an object called "final_table"
# this has 3.4 million rows...so you will need to be thoughtful about
# how you analyze the data so that you don't overwhelm your laptop
load("output/final_compiled_counts/joined_count_data.RData")

# changes all columns names to lower case so Travis doesn't complain
names(final_table) %<>% tolower

# test that it loaded correctly before proceeding
stopifnot(exists("final_table"))
```



```{r make-summary-table-SNOR-top15-genes}
# here I want to pull out just the top 15 genes that either 
#  SNORA- or SNORD- in their base name 
# have to filter it out as a subset from final_table
snor_genes <- final_table %>%
  filter(grepl("SNOR", genename)) %>%
  group_by(genename) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count)) %>%
  head(15)

# then we can use the `kable()` function to make a nicely formatted
# markdown table
snor_genes %>%
  kable()
```
**Table 1**: The most highly expressed SNOR-family genes across all patients of both genders.

Out of a total of 297 SNOR-family genes identified in patient lung tissue samples, the breakdown between the fifteen most represented by average sequence count was not quite even. There were nine SNORD- genes to only six SNORA-genes. Total counts per type of SNORA- and SNORD-genes were 129 and 168 respectively.

 
```{r make-all-SNOR-table, echo = FALSE}
# There are many many genes in this dataset, so we can
# subset it down to just a few here to look for 
# patterns in the most highly expressed
all_snor <- final_table %>%
  filter(grepl("SNOR", genename)) %>%
  group_by(gender, genename, smoking_status, cancer_stage) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count)) %>%
  head(n = 15)
# limiting to 15 just so that the results can be
# more easily visualized

```



```{r barplot-of-highly-expressed-genes-gender, echo = FALSE}
# this code uses the same data as above, but use it to make a
# barplot by gender of patients
all_snor %>%
  ggplot(aes(x = genename,
             y = mean_count,
             fill = gender)) +
    geom_col(position = "dodge")
```
**Figure 1**: Highest expressed SNOR-genes by gender representation.

When the SNOR-genes with the greatest mean count within the experimental group are categorized by gender, there is a strong divide along the gender line. Based on the nine highest expressed SNOR-genes visualized in Fig.1, only one of them had the highest rate of expression in females. The other eight had higher expression in males, and were exclusively being expressed in males for seven. The SNORD78 gene had the hightest mean count at an almost thousand count average.

```{r make-barplot-of-SNOR-smoking-status, echo = FALSE}
# this code uses the same data as above, but use it to make a
# barplot - remember geom_col() is just like
# when one would use geom_bar(stat = "identity")
all_snor %>%
  ggplot(aes(x = genename,
             y = mean_count,
             fill = smoking_status)) +
    geom_col(position = "dodge")
```
**Figure 2**: Highest expressed SNOR-genes by smoking status of patients.

The homogeneity seen when grouping the most expressed SNOR-genes by gender is much stronger than when expression levels are categorized by smoking status of the patients. Patients that were currently smoking at the time of the study had five SNOR-genes being highly expressed relative to other groups of other status, with the highest mean counts of all genes. Previous smokers had equivalent rates of overexpressing SNOR-genes, with five genes being overexpressed within the group, and the third highest mean count across all statuses. Patients whom had reportedly never smoked or had no history collected were only represented in two of the SNOR-genes with the highest mean counts. The gene SNORD44 had most variety for representation by smoking status, with current, previous, and non-smokers all having high expression rates.

```{r make-barplot-of-SNOR-smoking-status, echo = FALSE}
# this code uses the same data as above, but use it to make a
# barplot - remember geom_col() is just like
# when one would use geom_bar(stat = "identity")
all_snor %>%
  ggplot(aes(x = genename,
             y = mean_count,
             fill = cancer_stage)) +
    geom_col(position = "dodge")
```
**Figure 3**: Highest expressed SNOR-genes by cancer stage of patients.

When the stage of cancer at the time of tissue sampling is considered, the trends run almost parallel to when the genes were grouped by smoking status. However, the genes which were most expressed by current smokers were largely only at early stages of lung cancer (1A, 1B or 2B). The patients at the latter cancer stages (3A-4), did not exhibit the highest mean counts across genes. They did appear in six of the nine genes though, making of the majority of overexpressed SNOR-genes. Again, the SNORD44 gene showed great variablity in cancer stage as it did when grouped by smoking status. It had one of the highest mean counts for four stages of cancer, while SNORD78 followed closely with three stages  represented.


```{r sort-out-all-snor-genes}
# take out just snor family genes from final table
# for count of total snor-genes represented
final_snor <- final_table %>%
  filter(grepl("SNOR", genename)) %>%
  group_by(genename) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count)) %>%
  ungroup() %>%
  select(genename) %>%
  unique() %>%
  pull()
```

```{r sort-out-SNORA-genes}
# this will pull out just SNORA-genes
# for total count represented in patients
snora_genes <- final_table %>%
  filter(grepl("SNORA", genename)) %>%
  group_by(genename) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count)) %>%
  ungroup() %>%
  select(genename) %>%
  unique() %>%
  pull()
```

```{r sort-out-SNORD-genes}
# this will pull out just SNORD-genes
# for total count represented in patients
snord_genes <- final_table %>%
  filter(grepl("SNORD", genename)) %>%
  group_by(genename) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count)) %>%
  ungroup() %>%
  select(genename) %>%
  unique() %>%
  pull()
```
# Discussion

Add around 1-2 pages interpreting your results and considering future directions one might take in analyzing these data.


"gene amplification and the consequent overexpression of the amplified oncogene play an important role in the development of tumors, because their overexpression confers a growth advantage. The ability to identify putative oncogenes that are activated during tumorigenesis could facilitate the choice of molecular genetic targets for diagnosis and therapy of the disease." (Li et al., 2006)
"Non-coding RNAs (ncRNAs) are important regulatory molecules involving in various physiological cellular processes. Alterations of ncRNAs, particularly microRNAs, play crucial roles in tumorigenesis. Accumulating evidence indicates that small nucleolar RNAs (snoRNAs), another large class of ncRNAs, are gaining prominence and more actively involved in carcinogenesis than previously thought. Some snoRNAs exhibit differential expression patterns in a variety of human cancers and demonstrate capability to affect cell transformation, tumorigenesis, and metastasis."
"Given their vital role in diverse cellular processes, defining differential expression patterns of cancer type-specific snoRNAs should exploit for the development of novel cancer biomarkers. Indeed, we and others have demonstrated that snoRNAs are present in a stable form and consistently measurable in blood plasma, sputum, and urine samples [206-210]. Therefore, snoRNAs possess the potential as fluid-based biomarkers for cancers."
snoRD33	Increased		NSCLC	[58]
snoRD66	Increased		NSCLC
[@mannoor2012small]

# Sources Cited
